File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/PlaylistFollowPrivacy.java
package com.timothy.spotifyarchitecture.retrofit.models;

import android.os.Parcel;
import android.os.Parcelable;

import com.google.gson.annotations.SerializedName;

public class PlaylistFollowPrivacy {
    @SerializedName("public")
    public Boolean is_public;

    public PlaylistFollowPrivacy() {
    }
}

File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/PlaylistFollowPrivacy.kt
package com.timothy.spotifyarchitecture.retrofit.models

import android.os.Parcel
import android.os.Parcelable

import com.google.gson.annotations.SerializedName

class PlaylistFollowPrivacy {
    @SerializedName("public")
    var is_public: Boolean? = false
}
